static/js/apps/explore_landing/topics.json

```diff
@@ -290,8 +317,8 @@
         ]
       }
     },
-    "education": {
-      "title": "Education",
+    "schools-housing-commute": {
+      "title": "Schools, housing, and commute",
```

Comment:
nit: do we capitalize Housing and Commute?

Reply:
that sounds good to me. added.